Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2241.cs

```diff
@@ -37,7 +37,11 @@ protected override void Init ()
 		[Ignore("Fails intermittently on TestCloud")]
 		public void ChangeOrientationCheckScroll ()
 		{
-			var isAndroid = RunningApp is AndroidApp;
+#if __ANDROID__
```

Comment:
Love the ifdefs when they cut out tests entirely from a platform. However, seems to me, if they just run differently on each platform then the existing idiom of `RunningApp is AndroidApp` seems cleaner than the ifdefs. I wouldn't hold up the PR for this of course. Just wondering if I'm missing something..